cloud_Azure/terraform/module/README.md

```diff
@@ -47,30 +57,25 @@ module kentik_azure_integration {
 
 ### Python and dependencies
 
-This module uses python to gather all NSG from Resource Group and expose it to terraform as external data source.
+This module uses python to gather all NSG from Resource Groups and expose it to terraform as external data source.
```

Comment:
Why cannot this information be collected using Terraform?

Reply:
Good question; it seems that the available providers don't expose such functionality